Traceback (most recent call last):
  File "ForwardRun_MC_SM.py", line 207, in <module>
    trace = GetTrace(PREFIX,0,optimal=False)
  File "../Utilities/newfun_sm.py", line 320, in GetTrace
    chain_idx = len(outlist[0])-12; chunck_idx = len(outlist[0])-9
IndexError: list index out of range
/var/spool/slurmd/job3752261/slurm_script: line 19: /home/users/yanlan: Is a directory
